<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildProfile</class>
 <widget class="QWidget" name="BuildProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BuildProfile</string>
  </property>
  <widget class="QLineEdit" name="inputLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Input file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileSelect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="framePreview">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>371</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="frameForward">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="frameBack">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>650</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
